rllib/algorithms/ppo/ppo.py

```diff
@@ -405,7 +406,8 @@ def training_step(self) -> ResultDict:
             # that we don't have to do this back and forth
             # communication between driver and the remote
             # trainer workers
-
+            is_module_trainable = self.workers.local_worker().is_policy_to_train
```

Comment:
this is converted to a function inside RW. I am using the callable here assuming it's done in the rollout worker. Later it should only be done in the Learner. But since for now we have the RW, we don't want to duplicate the logic that can cause regression.